payment_sepa_dd_payment_mode/controllers/main.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import PaymentPortal
+
+
+class PaymentPortalSEPADD(PaymentPortal):
+    def _validate_transaction_for_order(self, transaction, sale_order_id):
+        """Throws a ValidationError if IBAN is not correct"""
```

Comment:
this doesn’t describe what happens, does it?

Reply:
Set payment_mode for order.